avocado/core/job.py

```diff
@@ -441,24 +441,24 @@ def pre_tests(self):
         self._result_events_dispatcher.map_method('pre_tests', self)
 
     def run_tests(self):
-        mux = getattr(self.args, "mux", None)
-        if mux is None:
-            mux = multiplexer.Mux()
-        if not mux.is_parsed():   # Mux not yet parsed, apply args
+        variant = getattr(self.args, "xxx_variants", None)
```

Comment:
`xxx` or `XXX` is commonly used in programming for something that is ugly, dirty or badly implemented. See https://en.wikipedia.org/wiki/Comment_(computer_programming)#Tags

`x-`, is commonly used to prefix options which are experimental.

(and your case is none of the above, if I understood you correctly)